src/Filters/SimpleCollapsibleFilter.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+export default class SimpleCollapsibleFilter extends React.Component {
```

Comment:
I don't think you need the additional class.  You should be able to fix `CollapsibleFilter`